lib/smartsheet/client_behaviour.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Smartsheet.ClientBehaviour do
+  @callback get_sheet(String.t(), List) ::
```

Comment:
should be noted that the `List` 2nd argument is optional. idk if that will cause problems